TheJumpCloud/support
# Sa 2540 review recovery email pr

## Issues
* [SA-2540](https://jumpcloud.atlassian.net/browse/sa-2540) - SA-2540

## What does this solve?
Adds -recoveryEmail parameter to Get,Set, New-JCUser
## Is there anything particularly tricky?
N/a
## How should this be tested?
1.  Import the branch module: Import-Module ".../support/PowerShell/JumpCloud Module/JumpCloud.psd1"
2. Create new user with recovery email param: `New-JCUser -firstname test -lastname recovery -username testRecoveryuser -email test@testdelete.com -recoveryEmail testRecoveryEmail@testdeleteme.com`
3.  `Get-JCUser -recoveryEmail testRecoveryEmail@testdeleteme.com`
4. `Set-JCUser -username testRecoveryuser -recoveryEmail newrecoveryemail@testdeleteme.com`
5. Lastly, verify: `Get-JCUser -recoveryEmail newrecoveryemail@testdeleteme.com`
